docs/developers/setup.md

```diff
@@ -93,11 +93,11 @@ With a real mnemonic, this would look like:
 export MNEMONIC="candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
 ```
 
-Once you've done that you're ready to run a truffle command. When using a mnemonic, your network argument should look
+Once you've done that you're ready to run a hardhat command. When using a mnemonic, your network argument should look
 like `--network [NETWORK_NAME]_mnemonic`. So, for example, using a mnemonic on Kovan would look like:
 
 ```bash
-yarn truffle console --network kovan_mnemonic
+npx hardhat console --network kovan_mnemonic
 ```
 
 ### Hardware wallets (more secure)
```

Comment:
I think we should consider removing the hardware wallet section since these commands do not work with hardhat.